src/Components/Web.JS/src/Boot.WebAssembly.ts

```diff
@@ -61,3 +63,19 @@ if (shouldAutoStart()) {
     }
   });
 }
+
+async function intiializeConfigAsync(platform: Platform, resourceLoader: WebAssemblyResourceLoader) : Promise<void> {
+  const configFiles = resourceLoader.readConfigFilesAsync();
```

Comment:
Likewise, it would be good if `WebAssemblyResourceLoader` didn't have special logic related to config, since it's meant to be a general resource loading system and agnostic to the kinds of resources it loads.

Could the `readConfigFilesAsync` logic go into whatever new file contains the logic to do with config?